File: java/210524_class2/src/com/lg/studApp/model/RandData.java
package com.lg.studApp.model;

import java.util.Random;

public class RandData {
	
	String name[] = { "박효신", "한효주", "한지민", "이승기", "차은우", "송중기" };
	int age[] = { 41, 46, 39, 38, 26, 29 };
	char gender[] = { '남', '여' };
	String address[] = { "대구시 동구 신천 1동", "대구시 동구 신천2동", "대구시 동구 신천3동", 
			"대구시 동구 신천4동", "대구시 동구 신천5동" };
	
	Random r = new Random();
	public String getName() {
		return name[r.nextInt(6)];
	}
	
	public int getAge() {
		return age[r.nextInt(6)];
	}
	
	public char getGender() {
		return gender[r.nextInt(2)];
	}
	
	public String getAddress() {
		return address[r.nextInt(5)];
	}

}

File: C#/210610/210610_class_student/210610_class_student/utill/RandData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _210610_class_student.utill
{
    class RandData
    {
        private string[] name = { "한지민", "박효신", "한효주", "박보영", "유아인" };
        private int[] age = { 20, 25, 30, 40, 35 };
        private char[] gender = { '남', '여' };
        private string[] address = { "대구 동구 동부로 13", "부산 해운대구 해변길 24", "서울 강남구 가로수길 15", "경남 함양군 유동길 16", "대구 달서구 감삼길 17" };
        Random r = new Random();

        public string getName()
        {
            return name[r.Next(0, 5)];
        }

        public int getAge()
        {
            return age[r.Next(0, 5)];
        }

        public char getGender()
        {
            return gender[r.Next(0, 2)];
        }


        public string getAddress()
        {
            return address[r.Next(0, 5)];
        }
    }
}
